tests/test_codex_feed_hooks.py

```
@@ -588,6 +615,1108 @@ def test_codex_monitor_survives_transient_owner_rpc_timeout(cli_path: str, root:
+            second_process = start_stop_hook()
+            second_outcome = fake.wait_for_frame(
+                lambda frame: "raw" in frame
+                or is_feed_event(frame, "SubagentStop"),
+                start_index=second_start,
+                timeout=3,
+            )
+            if second_outcome is None:
+                raise AssertionError(
+                    "The overlapping SubagentStop neither skipped the live "
+                    "claim nor exposed a duplicate replay"
+                )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)